internal/controller/databaseclusterrestore_controller.go

```diff
@@ -805,6 +805,11 @@ func (r *DatabaseClusterRestoreReconciler) tryCreatePG(ctx context.Context, obj
 		return err
 	}
 
+	if val, ok := pgRestore.GetAnnotations()[consts.ManagedByDataImportAnnotation]; ok && val == consts.ManagedByDataImportAnnotationValueTrue {
+		// part of data import, do not create a DBR.
+		return nil
+	}
+
```

Comment:
I don't get it why it is a problem that the controller creates a dbr when the upstream CR appears? Am I assuming that correctly that because of that confusing `Backup Source` field on the restores UI it was decided to get rid of the dbr CR and only keep the upstream CR?

Reply:
Clarified, resolving the comment. As discussed on the DS, we should skip creating for the importer restore because it's getting created twice - once by importer, second - as a response to the new upstream restore appearance.